### DISCUSSION TITLE
Ninite Integration.

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5884] [General]
**Is your feature request related to a problem? No

**Describe the solution you'd like**
I've been using MeshCentral for over a year, maybe close to 2 years and I've used Ninite for about 8-10 years. Ninite is a great way to install and keep programs updates plus its very simple to integrate with. Ninite.com

**Additional context**

**Comment by si458** [comment]
I don't understand how u can integrate ninte or why u want it integrated?
meshcentral is a remote support software, the updates are done inside the webui to upgrade meshcentral itself

**Comment by dinger1986** [comment]
If it's for installing can use winget natively with scripts or use an rmm

**Comment by ghost** [comment]
Integrating it would be simple. It uses an exe with args to install update or uninstall a whole list of programs and keeps them up to date. 

A lot of RMM platforms use this. Some like Pulseway I think list it as patch management. 

Just would be great to have a way to manage apps on computers to offer complete remote management.

**Reply by si458** [reply]
this wouldnt be something we add im afraid,
you should look into something like [tacticalrmm](https://github.com/amidaware/tacticalrmm) which is an RMM platform
meshcentral already has a run command feature, which you can load up powershell scripts to do things
and also plugins like [scripttask](https://github.com/ryanblenis/MeshCentral-ScriptTask) help extend automated scripts too
